text/0000-use-case-integration-best-practices.md

```diff
@@ -0,0 +1,177 @@
+- Feature Name: use_case_integration_best_practices
+- Status: Draft
+- Start Date: 2022-09-20
+- RFC PR: (leave this empty)
+- Hathor Issue: (leave this empty)
+- Author: Pedro Ferreira <pedro@hathor.network>
+
+# Summary
+[summary]: #summary
+
+This document presents best practices for use cases that integrate with the Hathor Network, in order to improve the reliability of their operation.
```

Comment:
The goal here is to present general best-practices or security best-practices?

In the first case, I think we might have some other best-practices that could be linked here, like this one which is still unmerged: https://github.com/HathorNetwork/rfcs/pull/30, or even something about the tx-mining-service.

But if we are focused on security here, maybe we should explicitly state this in the title and summary.

Reply:
Campana is now working on an integration guide for use cases. I feel we should focus this RFC in the security best practices and link them, as @msbrogli said. I will make this clear in the title and summary.